src/prx-theme.scss

```diff
@@ -0,0 +1,174 @@
+@import '~@angular/material/theming';
+
+@include mat-core();
+
```

Comment:
One thing I'm seeing strongly suggested here https://medium.com/@aleixsuau/how-to-use-angular-material-2-sass-variables-in-your-components-76ce0203f126 is defining the variables in a separate file so that we don't have to import mat-core and angular-material-theme every time we import prx-theme.scss from other files to access the variables.

Reply:
Great. I'll take a look at that article. Thanks!!!